sensor/tests/resource/helper.go

```diff
@@ -513,27 +571,45 @@ func createConnectionAndStartServer(fakeCentral *centralDebug.FakeService) (*grp
 	return conn, fakeCentral, closeF
 }
 
-// ApplyFileNoObject - apply a file without an object
-func (c *TestContext) ApplyFileNoObject(ctx context.Context, ns string, file YamlTestFile) (func() error, error) {
-	obj := objByKind(file.Kind)
-	return c.ApplyFile(ctx, ns, file, obj)
+// ApplyResourceNoObject - apply a resource without an object
```

Comment:
This comments can definitely be improved. I think they were added just to stop lint failures.

Here's my suggestion:
```
// ApplyResourceNoObject creates a Kubernetes resource using `ApplyResource` without requiring
// an object reference.

// ApplyResource creates a Kubernetes resource in namespace `ns` from a resource definition (see 
// `K8sResourceInfo` for more details). Once the resource is applied, the `obj` will be populated
// with the properties from the resource definition. In case the creation fails (due to the client
// API rejecting the definition), a `RetryCallback` function can be provided to manipulate the
// object prior to the retry.
```

Reply:
Agree. I'll update the comment